lib/modules/apostrophe-seo-doc-type-manager/index.js

```diff
@@ -14,6 +14,20 @@ module.exports = {
           label: 'Description',
           type: 'string',
           help: 'A short and accurate summary of the content of the page used in search results.'
+        },
+        {
+          name: '_seoCanonical',
+          label: 'Canonical Link',
+          type: 'joinByOne',
+          withType: 'apostrophe-page',
+          idField: 'pageId',
+          help: 'Is there a main version of this page that search engines should direct to?',
+          filters: {
+            projection: {
+              title: 1,
+              slug: 1
```

Comment:
`_url` should be included, though I think it might come through regardless. Good to have in there explicitly.